index.js

```diff
@@ -0,0 +1,22 @@
+const { parse, isValid } = require('./lib/input');
+const { execute } = require('./lib/notes');
+const minimist = require('minimist');
+
+console.log(minimist(process.argv.slice(2)));
+
+//bring execute, valid, and parse functions together.
+// use parse to create and object from process.argv.
+//use valid to check if it is a valid action.
+//if valid execute
+//IF NOT VALID show error.
+
+
+
+let parsed = parse(process.argv) ;
+
+if (isValid(parsed) === true) {
+    execute();
```

Comment:
you need to pass execute your parsed action:

```js
if(isValid(parsed)) execute(parsed);
else console.log('failure to start');
```

Reply:
Believe Completed